driver/src/main/java/org/kaazing/k3po/driver/internal/control/handler/ControlServerHandler.java

```diff
@@ -393,9 +457,25 @@ private void sendErrorMessage(ChannelHandlerContext ctx, Throwable throwable) {
             error.setSummary("Parse Error");
             Channels.write(ctx, Channels.future(null), error);
         } else {
-            logger.error("Internal Error. Sending error to client", throwable);
-            error.setSummary("Internal Error");
+            logger.error("Internal error. Sending error to client", throwable);
+            error.setSummary("Internal error");
             Channels.write(ctx, Channels.future(null), error);
         }
     }
+    
+    private void sendErrorMessage(ChannelHandlerContext ctx, String description) {
+        ErrorMessage error = new ErrorMessage();
+        error.setSummary("Internal error");
```

Comment:
Like in my answer [here](https://github.com/k3po/k3po/pull/390#discussion_r95751317) , I don't think this is a good idea. If you still think we should do this, let's discuss about it.